src/components/index.ts

```diff
@@ -28,7 +28,7 @@ export {Caption, CaptionProps} from './Caption';
 
 export {Card, CardProps} from './Card';
 
-export {Checkbox, CheckboxProps} from './Checkbox';
+export {Checkbox, CheckboxProps, CheckboxHandles} from './Checkbox';
```

Comment:
I'd like to avoid exporting this publicly, people shouldn't need it

Reply:
fair enough. I hade to disable strict-component-boundaries in the types file to do this.